### DISCUSSION TITLE
How to dont't update lazy-lock.json when installing dependencies

### DISCUSSION THREAD

**Discussion by fnoopv** [LazyVim/LazyVim#2237] [Q&A]
i have a nvim [config repo](https://github.com/fnoopv/nvim), used by lazyVim. i have two pc, when install dependencies lazy.nvim always update plugin version, that's not what i want it's should install dependencies strictly according to lazy-lock.json content?

**Comment by coder9194** [comment]
I also have the same problem, in my case, when I try to global restore (R) all versions of plugins to respect lock file, it always installs the latest version and overwrite my lock file.
